tests/integration/jpa/appl/NOTES.TXT

```diff
@@ -0,0 +1,56 @@
+JPA ISSUES IN NATIVE IMAGE WITH HIBERNATE
```

Comment:
Do you want to keep this file ? If yes please make that a .md file ; this could be a README.md

Reply:
I don't know yet. :) I can move it to an issue. But it will be better visible here.

Reply:
Actually, if this is tracking work items it's better to have this in an issue and use a task list.

Reply:
Moved to an issue.